app/services/record_participant_event.rb

```diff
@@ -20,23 +24,40 @@ def call
 
 private
 
-  def default_config
-    HashWithIndifferentAccess.new(
+  def initialize(params)
+    inject_params(params)
+  end
```

Comment:
That looks like we don't need it, do we?

Reply:
We don't. One of the comments was that if someone overrides the `initialize` and doesn't use `super` was that it wasn't clear what it was doing. I wanted to show it, but I was pretty sure that I'd get a comment like yours.